packages/itwin/grouping-mapping-widget/src/widget/GroupingMappingWidget.tsx

```diff
@@ -4,20 +4,29 @@
 *--------------------------------------------------------------------------------------------*/
 import type {
   AbstractWidgetProps,
-  UiItemsProvider} from "@itwin/appui-abstract";
+  UiItemsProvider,
+} from "@itwin/appui-abstract";
 import {
   AbstractZoneLocation,
   StagePanelLocation,
   StagePanelSection,
   StageUsage,
 } from "@itwin/appui-abstract";
-
-import * as React from "react";
+import type { AccessToken } from "@itwin/core-bentley";
+import React from "react";
 import GroupingMapping from "./components/GroupingMapping";
 
 export class GroupingMappingProvider implements UiItemsProvider {
   public readonly id = "GroupingMappingProvider";
 
+  private readonly _accessToken?: AccessToken;
+  private readonly _prefix?: "" | "dev" | "qa";
+
+  constructor(accessToken?: AccessToken, prefix?: "" | "dev" | "qa") {
```

Comment:
Should this be a prefix or would taking the full URL be better to future proof it?

Reply:
Yes, I originally planned on doing the full URL but imodel-browser-react does not allow this.

Reply:
The new upcoming reports widget will support full url and eventually One Click LCA widget when we get to it.

Reply:
okay, fine by me.